evm-tests/ethjson/src/test_helpers/state.rs

```diff
@@ -79,30 +76,12 @@ pub struct MultiTransaction {
 
 	/// EIP-4844
 	#[serde(default)]
-	pub blob_versioned_hashes: Vec<H256>,
+	pub blob_versioned_hashes: Vec<U256>,
```

Comment:
Just out of curiosity. Why did you change `H256` to `U256` here? As I remember, the `H256` type is used for hash storing.